tls/client-tls-nonblocking.c

```diff
@@ -33,37 +33,92 @@
 /* wolfSSL */
 #include <wolfssl/options.h>
 #include <wolfssl/ssl.h>
-#include <wolfssl/test.h>
 
 #define DEFAULT_PORT 11111
 
+#define CONNECT_WAIT_MS 2
+
 #define CERT_FILE "../certs/ca-cert.pem"
 
+enum {
+    TEST_SELECT_FAIL,
+    TEST_TIMEOUT,
+    TEST_RECV_READY,
+    TEST_SEND_READY,
+    TEST_ERROR_READY
+};
+
+
+static WC_INLINE int tcp_select(SOCKET_T socketfd, int to_sec)
```

Comment:
Please remove WC_INLINE. Looks like `SOCKET_T` comes from wolfssl/wolfio.h, so might be good to include that at top for clarity.